x-pack/plugins/task_manager/server/task_store.ts

```diff
@@ -340,6 +343,30 @@ export class TaskStore {
     return savedObjectToConcreteTaskInstance(result);
   }
 
+  /**
+   * Gets tasks by ids
+   *
+   * @param {Array<string>} ids
+   * @returns {Promise<ConcreteTaskInstance[]>}
+   */
+  public async bulkGet(ids: string[]): Promise<BulkGetResult> {
```

Comment:
New function that fetches the latest version of documents.. I couldn't use `search(...)` in a performant manner because I would have to wait for a document refresh before getting the new version of a document whenever a 409 error occurred.